src/client.ts

```diff
@@ -2150,4 +2153,31 @@ export class StreamChat<
   deleteBlockList(name: string) {
     return this.delete<APIResponse>(`${this.baseURL}/blocklists/${name}`);
   }
+
+  exportChannels(request: Array<ExportChannelRequest>) {
```

Comment:
this is a server only feature? should we throw error when it's called using client tokens?

Reply:
yes